projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -96,6 +103,10 @@ public function should_disable_links_manager() {
 	 * Adds My Home menu.
 	 */
 	public function add_my_home_menu() {
+		if ( self::CLASSIC_VIEW === $this->get_preferred_view( 'index.php' ) ) {
+			return;
+		}
+
 		$this->update_menu( 'index.php', 'https://wordpress.com/home/' . $this->domain, __( 'My Home', 'jetpack' ), 'manage_options', 'dashicons-admin-home' );
```

Comment:
It seems we will still need an update in order to change the Menu name to `My Home` instead of `Dashboard`

![SS 2021-06-24 at 13 01 09](https://user-images.githubusercontent.com/12430020/123244232-6104b180-d4ec-11eb-98fd-b56c41b057f6.png)

Reply:
We're finally excluding the quick switcher from My Home: pd2C0z-3q-p2#comment-146